### ISSUE TITLE
Element.Geometry returns null

### ISSUE THREAD

**Issue by paoloemilioserra** [DynamoDS/DynamoRevit#1470]
If this issue is **not** a bug report or improvement request, please check the [Dynamo forum](https://forum.dynamobim.com/), and start a thread there to discuss your issue.

## Dynamo version

1.2.1.3083

## Revit version

2017

## Operating system

W7

## What did you do? 
I have a Revit document that contains a floor with a slope arrow defined with a sketch that contains splines. **I think the RuledSurfaces are causing the error not being properly converted to ProtoTypes.**

Select Model Element > Element.Geometry
Select Model Element > Element.Solids
Select Model Element > Element.Faces

## What did you expect to see?

the geometry associated to the floor

## What did you see instead?

![image](https://cloud.githubusercontent.com/assets/14221455/22123406/c557a28e-de8b-11e6-87f6-3b22ee9fce16.png)

**Comment by mjkkirschner** [comment]
@paoloemilioserra yes interesting, thanks for the repro case we'll need the rvt file with this geometry.

**Comment by mjkkirschner** [comment]
@paoloemilioserra yes interesting, thanks for the repro case we'll need the rvt file with this geometry.

**Comment by paoloemilioserra** [comment]
[Test_FloorWithSplines.zip](https://github.com/DynamoDS/DynamoRevit/files/718751/Test_FloorWithSplines.zip)

**Comment by ZiyunShang** [comment]
I think this issue could be related to #1960 ,
this may be due to the limitations of the Dynamo Geometry kernel.

**Comment by johnpierson** [comment]
Closing this out. This is a known limitation in converting certain Revit geometry into Dynamo's geometry kernel rather than a defect in the node itself, and the report is many years old.

If you hit this on a current build, please open a fresh issue with the model attached so we have a current case to work from.
